composer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "dvsa/mot-doctrine-module",
+    "description": "DVSA Doctrine Module",
+    "type": "module",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Kainos",
+            "homepage": "https://www.kainos.com"
+        }
+    ],
+    "require": {
+        "php": "^8.0",
+        "laminas/laminas-servicemanager": "^3.10.0",
+        "doctrine/cache": "1.4.2",
+        "laminas/laminas-dependency-plugin": "^2",
+        "captainhook/captainhook": "^5.16",
+        "captainhook/plugin-composer": "^5.3"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.3"
+    },
+    "autoload": {
+        "psr-4": {
+            "DvsaDoctrineModule\\": "src/"
+        }
+    },
+    "scripts": {
+        "test": [
+            "phpunit"
+        ]
+    },
+    "config": {
+        "allow-plugins": {
+            "laminas/laminas-dependency-plugin": true,
+            "captainhook/plugin-composer": true
+        }
+    }
+}
```

Comment:
Changed: renamed package name and updated doctrine/cache package vulnerability caught on snyk